doc/rados/operations/operating.rst

```diff
@@ -352,7 +352,26 @@ To stop a Ceph daemon on another node, use the following syntax::
 	sudo service ceph -a stop osd.0
 
 
+Stopping an Entire Cluster
+==========================
+
+To stop an entire ceph cluster (for example, if you must physically move
+hosts in a data center) first set the noout flag to prevent OSDs from being
+removed::
+	
+	ceph osd set noout
+
+Then shutdown all the services in your cluster in the manner appropriate for your platform.
 
+.. note:: Ceph is known to complain in small ways when coming up after a complete shutdown.
+	It's behaviors in this case are not well validated at this point. Nothing should
+	go really wrong, but there may be some irritations associated with this operation.
+	After starting back up, if anything seems amiss, restart the daemons that are
+	misbehaving and their behavior should return to normal.
```

Comment:
i'm curious what those irritations are?  i can't think of anything that should work here...